fud/pyproject.toml

```diff
@@ -19,7 +19,8 @@ requires = [
   "packaging",
   "numpy",
   "simplejson",
-  "networkx>=2.6"
+  "networkx>=2.6",
+  "pynq"
```

Comment:
Move this into the `extra.fpga` requirements so that it isn't installed by default

Reply:
Also, if `pyopencl` is no longer used, remove it from the dependencies below